### PR TITLE
Newsletters: Update style and markup for intro screen

### LINKED ISSUE #77309: Newsletter onboarding: update LP
We should update the LP design to match the Marketing LP.

Current:

Image

New (note that the Jetpack logo is in the same position as the other newsletter onboarding steps, it shouldn't bounce from screen to screen):

Image

[Handoff and specs](HSljkKnYECEEegL3DqP2xC-fi-8815_84811)

### PR DESCRIPTION
**Pull request by edanzer**
Resolves https://github.com/Automattic/wp-calypso/issues/77309

## Proposed Changes

Update overall appearance and markup of the Newsletter intro screen in onboarding. Changes include: 
   - Changing background image
   - Showing header with Newsletter title
   - Updating title / description / button font
   - Adding button icon
   - Moving Jetpack logo from top to bottom
   
**Before**
intro-old

**After**
intro-new

## Testing Instructions

Load this branch locally, start a newsletter site at http://calypso.localhost:3000/setup/newsletter/intro, and confirm new design matches the above screenshot.

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx [hunk 1]
```diff
@@ -86,11 +86,11 @@ const useIntroContent = ( flowName: string | null ): IntroContent => {
 
 		if ( flowName === NEWSLETTER_FLOW ) {
 			return {
-				title: __( 'The newsletter. Elevated.' ),
+				title: __( 'Write. Grow. Earn. This is Newsletter.' ),
 				text: __(
-					'Everything you need to reach and grow an audience, with the power and permanence of WordPress.com.'
+					'Unlimitd subscribers. Beauiful design. And everything you need to grow your audience. Powered by WordPress.com'
```
Comment (lezama) [inline client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx]:
```suggestion
					'Unlimited subscribers. Beautiful design. And everything you need to grow your audience. Powered by WordPress.com'
```

Comment (edanzer) [reply client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx]: Going a bit to fast :). Thanks and fixed @lezama!

**Review by lezama — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8071071

Thank you @edanzer for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8071071).

**Pull request merged**